hera_cal/lstbin.py

```diff
@@ -267,19 +302,45 @@ def lst_bin(data_list, lst_list, flags_list=None, dlst=None, begin_lst=None, lst
 
             # take bin average: real and imag separately
             if median:
-                real_avg.append(np.nanmedian(d.real, axis=0))
-                imag_avg.append(np.nanmedian(d.imag, axis=0))
+                # for median to account for varying nsamples, copy each day by nsamps
```

Comment:
AFAIK we don't really use the median option. I'd be fine not supporting both `median==True` and `nsamp_list!=None` as long as that is documented.

Reply:
i've removed median support for non-None nsamples_list.